ext/curl/tests/bug72189.phpt

```diff
@@ -0,0 +1,24 @@
+--TEST--
+Bug #72189 (Missing CURL_VERSION_* constants)
+--SKIPIF--
+<?php
+include 'skipif.inc';
+
+$curl_version = curl_version();
+if ($curl_version['version_number'] < 0x070a06) {
+    exit('skip: test works only with curl >= 7.10.6');
+}
+
+?>
+--FILE--
+<?php
+
+echo CURL_VERSION_DEBUG."\n";
+echo CURL_VERSION_GSSNEGOTIATE."\n";
+echo CURL_VERSION_NTLM."\n";
```

Comment:
We shouldn't test specific values of constants, libcurl probably does not guarantee that these values are stable.

Generally I'd say that this kind of change is okay without tests.

Reply:
Fine. Test removed.